scripts/install

```diff
@@ -91,30 +98,17 @@ ynh_system_user_create --username=$app
 #=================================================
 # PHP-FPM CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring php-fpm..." --weight=2
+ynh_script_progression --message="Configuring PHP-FPM..." --weight=2
 
-# Create a dedicated php-fpm config
+# Create a dedicated PHP-FPM config
 ynh_add_fpm_config
 
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 # AUTOCONFIG
 #=================================================
-ynh_script_progression --message= "Configuring rainloop..." --weight=3
-
-# Set lang => define from install manifest
-case "$lang" in
```

Comment:
Idk, we never use this variable after installing this app, but we store it. And we stored it with the old values `English` or `French`. If we want to use it later (which is not the case), we should update this value in the upgrade script. Or we could consider that we don't use it, so we can also not store this value...

Reply:
ugh yeah, now that you mention it it seems to never be used x_x Quite weird from a UX point of view to have this question during install then x_x

Reply:
Well, in fact it is, but only during the install

Reply:
> And we stored it with the old values `English` or `French`

O.o  The stored value for `$lang` should be the one proposed in the manifest: `en`, `fr` ... no?

Reply:
Yes indeed, and as you changed the possible values, we should migrate it into the upgrade script too to avoid any problem later.